scripts/player.gd

```diff
@@ -3,22 +3,25 @@ var velocity : Vector2 = Vector2(0,0)
 var up : Vector2 = Vector2(0,-1)
 var health : float = 100.0
 
+var walkspeed = 150
```

Comment:
camelCase for variable names please.

Rename to walkSpeed

Also might be better to export the variable

export var walkSpeed

That way u can just modify it on the Editor and not have to dig thru the code each time